packages/atomic/cypress/e2e/result-list/result-components/result-multi-value-text.cypress.ts

```diff
@@ -49,260 +47,150 @@ export interface FacetWithResponseProps {
-
-        describe('with max-values-to-display=4', () => {
-          const rawValues = Object.keys(values);
-
-          describe('by default', () => {
-            beforeEach(() => {
-              prepareValuesWithMaximum(values, 4).init();
-            });
-
-            assertShouldRenderValues(localizedValues);
-            assertDoesNotDisplayXMoreLabel();
-          });
-
-          describe('with slots', () => {
-            const buildSlotElement = (value: string, text: string) => {
-              const el = generateComponentHTML('span', {
-                slot: `result-multi-value-text-value-${value}`,
-              });
-              el.innerText = text;
-              return el;
-            };
+      new TestFixture()
+        .with(
+          addMultiValueText(
+            {
+              field,
+              'max-values-to-display': 4,
+            },
+            slotElement
+          )
+        )
+        .with(addFieldValueInResponse(field, originalValues))
+        .withFieldCaptions(field, values)
+        .withTranslation({'n-more': '{{value}} more'})
+        .init();
 
-            beforeEach(() => {
-              prepareValuesWithMaximum(values, 4, [
-                buildSlotElement(rawValues[1], 'A'),
-                buildSlotElement(rawValues[3], 'B'),
-              ]).init();
-            });
+      assertShouldRenderValues([
+        localizedValues[0],
+        'The field value in at position 1 should be replaced by this',
+        localizedValues[2],
+      ]);
+    });
 
-            assertShouldRenderValues(
-              [localizedValues[0], 'A', localizedValues[2], 'B'],
-              'should replace the correct values'
-            );
-          });
+    it('with a facet and two selected values, it should display the selected values first', () => {
+      const selectedValues = [originalValues[1], originalValues[2]];
 
-          describe('with a facet and two selected values', () => {
-            const facetId = 'blah';
-            const selectedValues = [rawValues[1], rawValues[2]];
-            beforeEach(() => {
-              prepareValuesWithMaximum(values, 4)
-                .with(
-                  addFacetWithResponse({
-                    facetId,
-                    field,
-                    values: rawValues,
-                    selectedValues,
-                  })
-                )
-                .init();
+      new TestFixture()
+        .with(
+          addMultiValueText({
+            field,
+            'max-values-to-display': 4,
+          })
+        )
+        .with(addFieldValueInResponse(field, originalValues))
+        .withFieldCaptions(field, values)
+        .withTranslation({'n-more': '{{value}} more'})
+        .with((fixture: TestFixture) => {
+          fixture
+            .with(addFacet({'facet-id': 'blah', field}))
+            .withCustomResponse((response) => {
+              response.facets = [
+                {
+                  facetId,
```

Comment:
I think you forgot to intialize this facetId variable. I am getting errors in cypress and vscode